LayoutTests/fullscreen/full-screen-test.js

```diff
@@ -158,7 +158,25 @@ function waitForEventTestAndEnd(element, eventName, testFuncString)
 {
     waitForEventAndTest(element, eventName, testFuncString, true);
 }
-  
+
+function waitFor(element, type, silent, success) {
+    return new Promise(resolve => {
+        element.addEventListener(type, event => {
+            if (silent) {
+                resolve(event);
+                return;
+            }
+
+            if (success !== undefined)
+                logResult(success, `EVENT(${event.type})`);
+            else
+                consoleWrite(`EVENT(${event.type})`);
+            
+            resolve(event);
+        }, { once: true });
+    });
+}
+
```

Comment:
This will just override the `waitFor` function above and this doesn't seem to be used in this patch?

Reply:
Ah! I missed that I'd already imported `waitFor()` from video-test.js.